[inductor] fix TritonTemplateCaller.__str__

shunting314 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #97578

We remove TritonTemplateCaller.to_callable previously. But this method is still used in `TritonTemplateCaller.__str__` . The to_callable method in the base class will be used and raise an exception.

This PR fix TritonTemplateCaller.__str__ to return the string representation without calling to_callable.


cc @soumith @voznesenskym @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

shunting314 committed:
[inductor] fix TritonTemplateCaller.__str__

[ghstack-poisoned]

nmacchioni reviewed (approved):
LGTM, thanks for the fix!

shunting314 commented:
@pytorchbot merge

shunting314 committed:
Update on "[inductor] fix TritonTemplateCaller.__str__"


We remove TritonTemplateCaller.to_callable previously. But this method is still used in `TritonTemplateCaller.__str__` . The to_callable method in the base class will be used and raise an exception.

This PR fix TritonTemplateCaller.__str__ to return the string representation without calling to_callable.


cc soumith voznesenskym penguinwu anijain2305 EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 desertfire

[ghstack-poisoned]

shunting314 commented:
@pytorchbot merge -f "segfault is unrelated"
